otherlibs/dynlink/byte/dynlink.ml

```diff
@@ -39,7 +39,11 @@ module Bytecode = struct
         @ Symtable.required_globals t.cu_reloc
       in
       let required =
-        List.filter (fun id -> not (Ident.is_predef id)) required
+        List.filter
+          (fun id ->
+             not (Ident.is_predef id)
+             && not (String.contains (Ident.name id) '.'))
```

Comment:
Without knowing the context, this looks a bit strange. Where is the code that creates predefined identifiers with `.` in their name?

Reply:
(Well I guess those are the `Ident.create_persistent (packagename ^ "." ^ name)` calls in `bytecomp/bytepackager.ml`.

Reply:
Yeah, and you'll also notice a use of `String.contains ... '.'` in that file as well. It's not pleasant, but there are a couple of PRs coming down the pipeline that will tidy up this stuff so I'm not in a hurry to fix it right now.